### PR TITLE
Apply missing pretty formats

### PR DESCRIPTION
**Pull request by stefnestor**
Adds in missing columns, pretty json, & sorting onto existing request uri's.

### CODE REVIEW

File: src/main/resources/elastic-rest.yml [hunk 1]
```diff
@@ -139,7 +139,7 @@ alias:
 
 allocation:
   versions:
-    ">= 0.9.0": "/_cat/allocation?v&format=json"
+    ">= 0.9.0": "/_cat/allocation?v&pretty&format=json"
```
Comment (pickypg) [inline src/main/resources/elastic-rest.yml:142]: Have you validated that these work with older versions?

**Comment by inqueue** [comment]
My vote is for less `?pretty` since it adds extra overhead to diagnostic collection and bundle size. +1 to removing it whenever we can.

File: src/main/resources/elastic-rest.yml (hunk 1 above)
Comment (stefnestor) [reply src/main/resources/elastic-rest.yml:142]: I did not 😞 - AFAICT `pretty` is used / ignored but doesn't cause calls to fail. Is that incorrect?

**Comment by stefnestor** [comment]
> My vote is for less ?pretty since it adds extra overhead to diagnostic collection and bundle size. +1 to removing it whenever we can.

@inqueue, do we have an alternative way of formatting all JSON on save, because agreed that would be preferred. This is just bringing formatting to expected par through current method.

**Comment by inqueue** [comment]
> > My vote is for less ?pretty since it adds extra overhead to diagnostic collection and bundle size. +1 to removing it whenever we can.
> 
> @inqueue, do we have an alternative way of formatting all JSON on save, because agreed that would be preferred. This is just bringing formatting to expected par through current method.

I don't other than the consumer using `jq` or other tools to prettify. The problem is we don't know when pretty json is going to break a tool, https://github.com/elastic/support-diagnostics/pull/468 for example.

API call output size for those in this PR are typically nowhere near what we might see in a cluster state. My comment was aimed at keeping consistency in data collection across all API calls, but it can come at a cost of interrupting workflow, requiring additional tools, etc. I'm fine with adding pretty to these calls...or not :)

**Comment by jpcarey** [comment]
my personal opinion is to not have the `?pretty` flag. I think we should do our best to reduce our footprint at collection time (size, memory, etc). Having a simple tool / script to "extract" the data on our side could take care of prettifying the json, and we could reduce the number of API calls (eg. the cluster state contains the templates and mappings, why fetch those separately).

**Comment by stefnestor** [comment]
>  I think we should do our best to reduce our footprint at collection time (size, memory, etc). Having a simple tool / script to "extract" the data on our side could take care of prettifying the json

@jpcarey IMO "on our side" should still be within the diagnostic code and not a separate Elastic tool, so that the original diagnostic pull is as helpful as possible to the widest audience. I agree on having a lighter footprint & believe switching out `?pretty` for a simple JSON formatting code would be extremely light within the diagnostic itself & lighter than it currently is.

This would still allow the diagnostic to be usable by Elastic Support as well as Elastic users & not restrict widespread audience usage or require a tool outside the diagnostic itself to digest it. I'm not familiar w/how to add separate sub-code paths in terms of this repo, so I'll close this PR & file as an issue. Thanks!

**Pull request closed without merging**